schedule/yast/yast_hostname/yast_hostname+linuxrc_hostname_aarch64.yaml

```diff
@@ -0,0 +1,22 @@
+---
```

Comment:
please apply similar change requested in `schedule/yast/yast_hostname/yast_hostname+dhcp_hostname_aarch64.yaml`